frontend/packages/client/src/pages/ProposalCreate.js

```diff
@@ -75,6 +75,19 @@ export default function ProposalCreatePage() {
 
     const body = customDraftToHTML(currentContent);
 
+    const hasValidStartTime = isStartTimeValid(
+      stepsData[1].startTime,
+      stepsData[1].startDate
+    );
+
+    if (!hasValidStartTime) {
+      notifyError({
+        status: 'Invalid start time for proposal',
+        statusText: 'Please update start time on Set Date & Time step',
+      });
+      return;
+    }
+
```

Comment:
if user stays for a long time on the same create proposal step when creating the proposal time will be less than one hour. Adding this check will show this error message:

<img width="787" alt="Screen Shot 2022-08-26 at 11 58 59" src="https://user-images.githubusercontent.com/7526740/186941437-2220fb02-3ebb-4fa5-9e68-d13980e52d99.png">

Then user can go back to update start time date